llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -523,19 +586,23 @@ struct LoadedDebugInfoSection {
   Error addNode(Expected<MCObjectProxy> Proxy) {
     if (!Proxy)
       return Proxy.takeError();
-    if (auto CURef = DebugInfoCURef::Cast(*Proxy)) {
+    if (auto FixupsRef = DebugInfoFixupsRef::Cast(*Proxy)) {
+      if (CUData.size()) {
```

Comment:
single statement if's don't take `{}` (see the other examples in this function)

Reply:
Yep, my bad!